libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/DirectionsRouteEx.kt

```diff
@@ -0,0 +1,8 @@
+@file:JvmName("DirectionsRouteEx")
```

Comment:
This is probably not needed as we're accessing it from Kotlin.

Reply:
using for tests

Reply:
Java tests? :trollface:

Reply:
Jokes aside, I think we don't need it. Tests are Kotlin.

Reply:
it's useful for mocking static calls smt like `mockStatic("com.package.ClassName")`

Reply:
Are you sure? https://github.com/mapbox/mapbox-navigation-android/blob/716dcc5a9e704832a784bf99fd3736c02a0b1409/libnavigation-core/src/test/java/com/mapbox/navigation/core/trip/session/MapboxTripSessionTest.kt#L99 `NavigatorMapperKt` doesn't have it and it works 🤔

Reply:
agree we could remove it. But it doesn't make effect anyway